airflow-core/src/airflow/ui/src/layouts/Details/PanelButtons.tsx

```diff
@@ -223,7 +223,7 @@ export const PanelButtons = ({
   );
 
   return (
-    <Box position="absolute" ref={containerRef} top={1} width="100%" zIndex={1}>
+    <Box ml="-2px" position="absolute" px={4} ref={containerRef} top={1} width="100%" zIndex={1}>
```

Comment:
Also there's probably a way to get away with that without specifying negative margins which are usually a mess to deal with.